frontend/src/container/FeedItemWithPreview.jsx

```diff
@@ -286,6 +288,25 @@ export class FeedItemWithPreview extends React.Component {
     this.setState({ translationTargetLanguageCode })
   }
 
+  handleClickToggleComments = () => {
+    this.setState(previousState => ({
+      commentsShown: !previousState.commentsShown
+    }))
+  }
+
+  getCommentText = () => {
+    const { props, state } = this
+    if (props.commentList.length > 0) {
+      return state.commentsShown
+        ? `${props.t('Hide discussion')}`
+        : `${props.t('Show discussion')} (${props.commentList.length})`
+    } else {
+      return state.commentsShown
+        ? `${props.t('Hide')}`
+        : `${props.t('Comment')}`
```

Comment:
The new text is just a suggestion, if you apply, don't forget to change the translations. _Note: in portuguese "Hide comment area": "Ocultar área de comentário"._

```suggestion
        ? props.t('Hide comment area')
        : props.t('Comment')
```